Memoria.Launcher/Memoria/GameSettingsControl.cs

```diff
@@ -475,28 +396,13 @@ private void LoadSettings()
                 value = iniFile.ReadValue("Memoria", nameof(DownloadMirrors));
                 if (String.IsNullOrEmpty(value))
                 {
-                    if (true)
-					{
-                        // Use Github repository
-                        _downloadMirrors = new[]
-                        {
-                            "https://github.com/Albeoris/Memoria/releases/latest/download/Memoria.Patcher.exe"
-                        };
-                    }
-                    else if (CultureInfo.CurrentCulture.TwoLetterISOLanguageName == "ru")
+                    if (CultureInfo.CurrentCulture.TwoLetterISOLanguageName == "ru")
                     {
-                        _downloadMirrors = new[]
-                        {
-                            "https://ff9.ffrtt.ru/rus/FF9RU.exe",
-                            "https://ff9.ffrtt.ru/rus/Memoria.Patcher.exe"
-                        };
+                        _downloadMirrors = ["https://ff9.ffrtt.ru/rus/FF9RU.exe", "https://ff9.ffrtt.ru/rus/Memoria.Patcher.exe"];
```

Comment:
Why this change? Previously those mirrors would never be checked.

Reply:
Well Albeoris is Russian and I saw this code not being used so I supposed it might have been a mirror to allow non-vpn users...

Yeah maybe that's not very useful, we should always direct to the only up to date source

Reply:
I have not read everything, but these mirrors are the ones that were used prior to the latest release (Jun 11, 2023). I decided at that time to use this Github as a mirror but also to keep those lines in the code, even disabled, to keep their trace.